spec/system/task_spec.rb

```diff
@@ -1,58 +1,93 @@
 require 'rails_helper'
 RSpec.describe 'タスク管理機能', type: :system do
-  before do
-    FactoryBot.create(:task)
-    FactoryBot.create(:second_task)
-  end
   describe 'タスク一覧画面' do
     context 'タスクを作成した場合' do
       it '作成済みのタスクが表示される' do
+        create(:task)
         visit tasks_path
-        expect(page).to have_content 'test_title_01'
+        expect(page).to have_content 'test_title_1'
       end
     end
     context '複数のタスクを作成した場合' do
       it 'タスクが作成日時の降順に並んでいる' do
+        create_list(:task, 3)
```

Comment:
この処理より前にtaskを作成するテストケースが追加されると連番の値がズレてテストがすべて失敗してしまいます。

一度ローカル変数に入れて、タイトルの値を動的に取得させるようにしましょう。
```suggestion
        task_list = create_list(:task, 3)
```

Reply:
c3343ca　で修正しました。
動的にタイトルの値を取得させる(1ずつ増やす？)書き方がどうしてもわからなかったため、
taskとは別にmultipul_taskというfactoryBotを作り、利用しました。